src/main/scripts/src/app/shared/common.scss

```diff
@@ -5,3 +5,13 @@ a {
   cursor: pointer;
   transition: color $animation-color-duration;
 }
+
+ul {
+  list-style-type: none;
+}
+
+input {
+  border: none;
+  background: transparent;
+  outline: none;
```

Comment:
Do we want these styles in all components?

Reply:
Well it didn't break anything and most of the inputs we use are for editable names, (so they should look like regular text) not as much for forms.

Reply:
I would move these styles to the components where they are really needed. Moreover, try to define a class with these properties and apply it on elements where it is needed. It's better than overriding the default look of elements without styles.